akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/SerializationSupport.scala

```diff
@@ -73,7 +73,8 @@ trait SerializationSupport {
         readChunk()
     }
 
-    readChunk()
+    try readChunk()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/vasseurjb/akka/file/2046293339/issues/source?bid=3132306&fileBranchId=3132365#l76)